packages/cli/generators/openapi/schema-helper.js

```diff
@@ -171,11 +171,17 @@ function mapObjectType(schema, options) {
         const itemType =
           propertyType.itemType.kind === 'class'
             ? propertyType.itemType.className
-            : getJSType(propertyType.itemType.name);
+            : // The item type can be an alias such as `export type ID = string`
+              getJSType(propertyType.itemType.declaration) ||
+              // The item type can be `string`
+              getJSType(propertyType.itemType.name);
```

Comment:
Have you considered the following, to avoid duplication of `getJSType`?

```ts
: getJSType( 
  // The item type can be an alias such as `export type ID = string`
  propertyType.itemType.declaration ||
  // The item type can be `string`
  propertyType.itemType.name
)
```